internal/app/kdk/init.go

```diff
@@ -0,0 +1,141 @@
+// Copyright © 2018 Cisco Systems, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package kdk
+
+import (
+	"context"
+	"io/ioutil"
+	"os"
+	"os/user"
+	"path"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/cisco-sso/kdk/internal/pkg/utils"
+	"github.com/docker/docker/client"
+	"github.com/manifoldco/promptui"
+)
+
+var (
```

Comment:
This kind of looks like a struct, but it isn't :)

Reply:
should probably be a struct.